apps/common-app/src/examples/RuntimeTests/tests/animations/withSequence/arrays.test.tsx

```diff
@@ -0,0 +1,142 @@
+  finalValues: [number, number, number];
+  animationNumber: number;
+};
+
+describe('withSequence animation of number', () => {
+  const COMPONENT_REF = {
+    first: 'firstComponent',
+    second: 'secondComponent',
+    third: 'thirdComponent',
+  };
+
+  const DELAY = 50;
+  const WidthComponent = ({ startValues, middleValues, finalValues, animationNumber }: TestCase) => {
+    const lefts = useSharedValue<[number, number, number]>(startValues);
+    const ref0 = useTestRef(COMPONENT_REF.first);
+    const ref1 = useTestRef(COMPONENT_REF.second);
+    const ref2 = useTestRef(COMPONENT_REF.third);
+
+    const animateValueFn = useCallback(
+      function animateValue(finalValues: [number, number, number]) {
+        'worklet';
+        const finalValuesPlus20 = finalValues.map(val => val + 20);
+        switch (animationNumber) {
+          case 0:
+            return withSequence(
+              withTiming(finalValues, { duration: 200 }),
+              withDelay(DELAY, withTiming(middleValues, { duration: 300, easing: Easing.exp })),
+              withDelay(DELAY, withTiming(finalValuesPlus20, { duration: 200 })),
+            );
+          case 1:
+            return withSequence(
+              withSpring(finalValues, { duration: 200, dampingRatio: 1 }),
+              withDelay(DELAY, withSpring(middleValues, { duration: 300, dampingRatio: 1.5 })),
+              withDelay(DELAY, withSpring(finalValuesPlus20, { duration: 200, dampingRatio: 0.9 })),
+            );
+          case 2:
+            return withSequence(
+              withSpring(finalValues, { duration: 200, dampingRatio: 1 }),
+              withDelay(DELAY, withTiming(middleValues, { duration: 300 })),
+              withDelay(DELAY, withSpring(finalValuesPlus20, { duration: 200, dampingRatio: 1 })),
+            );
+        }
+        return [0, 0, 0];
+      },
+      [animationNumber, middleValues],
+    );
+
+    const style0 = useAnimatedStyle(() => {
+      return { left: lefts.value[0] };
+    });
+    const style1 = useAnimatedStyle(() => {
+      return { left: lefts.value[1] };
+    });
+    const style2 = useAnimatedStyle(() => {
+      return { left: lefts.value[2] };
+    });
+
+    useEffect(() => {
+      lefts.value = animateValueFn(finalValues) as [number, number, number];
+    });
+    return (
+      <View style={styles.container}>
+        <Animated.View ref={ref0} style={[styles.animatedBox, style0]} />
+        <Animated.View ref={ref1} style={[styles.animatedBox, style1]} />
+        <Animated.View ref={ref2} style={[styles.animatedBox, style2]} />
+      </View>
+    );
+  };
+
+  test.each([
+    { startValues: [0, 10, 20], middleValues: [0, 10, 20], finalValues: [20, 10, 30], animationNumber: 0 },
+    { startValues: [0, 10, 20], middleValues: [0, 50, 60], finalValues: [40, 10, 30], animationNumber: 1 },
+    { startValues: [30, 10, 55], middleValues: [0, -10, 60], finalValues: [40, 10, 30], animationNumber: 1 },
```

Comment:
Shouldn't this be `animationNumber: 2`?
Width component has cases for animation numbers: `0`, `1`, and `2`, but only `0` and `1` is used.
If it's intentional, is it necessary to have a `case` for `animationNumber = 2` in the `WidthComponent`?